manifests/i/Icarus/IcarusVerilog/12.2022.06.11/Icarus.IcarusVerilog.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Icarus.IcarusVerilog
```

```suggestion
PackageIdentifier: Icarus.Verilog
```
Brevity is preferred where possible